src/main/java/uk/gov/hmcts/reform/sscs/service/evidence/EvidenceUploadService.java

```diff
@@ -343,6 +344,7 @@ private void updateCaseDataWithNewPdfUpload(SscsCaseData sscsCaseData, LocalDate
         List<ScannedDocument> newScannedDocumentsList = union(emptyIfNull(sscsCaseData.getScannedDocuments()),
                 emptyIfNull(scannedDocuments));
         sscsCaseData.setScannedDocuments(newScannedDocumentsList);
+        sscsCaseData.setHasUnprocessedAudioVideoEvidence(YesNo.NO);
```

Comment:
Think this needs the same comment as Ryan put on mine yesterday: https://github.com/hmcts/sscs-tribunals-case-api/pull/2419 . You need this method to calculate the 'HasUnprocessedAudioVideoEvidence': setHasUnprocessedAudioVideoEvidenceFlag(sscsCaseData);

Reply:
Done